static/app/views/performance/traceDetails/index.tsx

```diff
@@ -79,21 +85,35 @@ class TraceSummary extends Component<Props> {
       error: QueryError | null;
       isLoading: boolean;
       meta: TraceMeta | null;
-      traces: TraceFullDetailed[] | null;
-    }) => (
-      <TraceDetailsContent
-        location={location}
-        organization={organization}
-        params={params}
-        traceSlug={traceSlug}
-        traceEventView={this.getTraceEventView()}
-        dateSelected={dateSelected}
-        isLoading={isLoading}
-        error={error}
-        traces={traces}
-        meta={meta}
-      />
-    );
+      traces: (TraceFullDetailed[] | TraceDetailedSplitResults) | null;
+    }) => {
+      let transactions: TraceFullDetailed[] | undefined;
```

Comment:
This is where the response split from just `{[transacations...]} `to `{transaction: [...], orphan_errors:[...]}` is handled when DETAILED TRACE is requested. Preserves functionality with or without flag.